src/patternfly/demos/Card/card-content-2.hbs

```diff
@@ -0,0 +1,24 @@
+{{#> card-header}}
+  {{#> l-flex flex--modifier="pf-m-align-center"}}
+    {{#> l-flex flex--modifier="pf-m-grow"}}
+      {{#> brand brand--attribute='src="/assets/images/activemq-core_200x150.png" width="60px" alt="Logo"'}} 
+      {{/brand}}
+    {{/l-flex}}
+    {{#> flex-item flex-item--modifier="pf-u-mr-0"}}
+      {{#> dropdown id=(concat card--id "-dropdown-kebab-right-aligned") dropdown--IsActionMenu="true" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+      {{/dropdown}}
+    {{/flex-item}}
+     <input type="checkbox" id="{{card--id}}-check" name="{{card--id}}-check" aria-labelledby="{{card--id}}-check-label">
+  {{/l-flex}}
+{{/card-header}}
+{{#> card-body}}
+  {{#> l-flex flex--modifier="pf-m-column"}}
+  <p style="font-size:18px" id="{{concat card--id "-check-label"}}">Patternfly</p>
+  <p style="font-size:14px;color:#8B8D8F;">Provided by Red Hat</p>
+  {{#> flex-item flex-item--modifier="pf-u-mt-md"}}
+    <p style="font-size:14px">
```

Comment:
what if you had for this section...

```
{{#> title}} Patternfly{{/title}}
{{#> content}}
    <small>provided by Red Hat</small>
    <p>The ActiveMQ component allows messages to be sent to a JMS Queue or Topic; or messages to be consumed from a JMS Queue or Topic using Apache ActiveMQ.</p>
{{/content}}
```

would that create the same spacing?

Reply:
We should also probably talk about whether we want to write styles for demos. I think a stylesheet for the demo is better than inline styles, if we're going to start writing custom CSS in demos that we don't support using default layouts/components.

Another option is to keep our demos within the bounds of what our components and layouts provide. If we don't currently have support for styles needed in a demo, we could 1) simplify the demo, or 2) expand our components/layouts so that we don't have to write custom CSS.

Reply:
Agreed I think it would be a great topic for CSS Army. To me, the demos showcase how our components fit together, and shouldn't need to add custom styles. I think custom styles are obvious and infinite, and we shouldn't need to worry about them in demos. So unless the design is necessary to the component, we don't add custom styles.

Reply:
Agreed. I had talked with @mceledonia about creating these demos and we mentioned creating more complex demos that contain custom CSS to show what a user could do (with these different styles in the cards), but I don't know if that's a good idea or not. I certainly like the idea of just building what we can without the need for custom CSS.

Reply:
I also think that if we add custom CSS, users will think that those styles are bundled with PF and it will create confusion as to what Patternfly really offers

Reply:
this^^. That's also why we don't include any JS in our demos. We used to include a little just for basic functionality but we don't want users to think we're suggesting or support that.